docs/DeveloperGuide.md

```diff
@@ -6,33 +6,230 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### The Overall UML Diagram:
 
+![UML Diagram](assets/images/mathGenius.png)    
+
+
+### The overall Sequence Diagram:
+![Sequence Diagram](assets/images/sequenceDiagram.png)    
```

Comment:
Could you perhaps break large sequence diagram into smaller, simpler sequence diagrams that can be placed in each component section>